bin/nodist

```diff
@@ -1,10 +1,16 @@
 #!/bin/sh
 
 if [ "$1" = "env" -o "$1" = "use" ]; then
-    \nodist + $2
-    if [ $? -eq 0 ]; then
-        export NODIST_VERSION=$2
+    "$0" + $2
+    if [ $? -eq 0 ]; then # if the last command exited with 0
+        export NODIST_NODE_VERSION=$2
+    fi
+else if ["$1" = "npm" -a "$2" = "env"]; then
+    "$0" npm + $3
+    if [ $? -eq 0 ]; then # if the last command exited with 0
+        export NODIST_NPM_VERSION=$3
     fi
 else
-    \nodist $*
-fi
\ No newline at end of file
+    DIR=$(echo "$NODIST_PREFIX" | sed -e 's,\\,/,g')
+    "$DIR/bin/node.exe" "$DIR/cli" "$@"
+fi
```

Comment:
You should remove everything from this file, except lines 14 and 15, then move the npm stuff to the nodist() function in nodist.sh